dogstatsd.py

```diff
@@ -485,6 +490,13 @@ def init(config_path=None, use_watchdog=False, use_forwarder=False, args=None):
     recent_point_threshold = c.get('recent_point_threshold', None)
     server_host = c['bind_host']
 
+    buffer_size = int(c.get('buffer_size', DEFAULT_BUFFER_SIZE))
+    if buffer_size in BUFFER_SIZE_OPTIONS:
+        buffer_size = buffer_size * 1024
+    else:
+        buffer_size = DEFAULT_BUFFER_SIZE * 1024
+        log.warning("Specified buffer size is invalid. Defaulting to %s." % (buffer_size))
```

Comment:
It's probably more readable if you say `log.warning("Specified buffer size is invalid. Defaulting to %s K." % (DEFAULT_BUFFER_SIZE))`, otherwise specify the units - ie. 8192 _bytes_.